components/HelpScreen.tsx

```diff
@@ -1,15 +1,41 @@
-import React, {useState} from 'react';
+import React, {useEffect, useState} from 'react';
 import {useTranslation} from 'react-i18next';
-import {Pressable} from 'react-native';
+import {Linking, Pressable} from 'react-native';
 import {Modal} from './ui/Modal';
 import {ScrollView} from 'react-native-gesture-handler';
 import {Column, Text} from './ui';
 import {Theme} from './ui/styleUtils';
+import getAllConfigurations from '../shared/commonprops/commonProps';
 
 export const HelpScreen: React.FC<HelpScreenProps> = props => {
   const {t} = useTranslation('HelpScreen');
 
   const [showHelpPage, setShowHelpPage] = useState(false);
+  const [hereUrl, setHereUrl] = useState('');
+
+  useEffect(() => {
+    getAllConfigurations().then(response => {
+      setHereUrl(response.aboutInjiUrl);
+    });
+  }, []);
+
+  var diffTypesOfIdsUrl = hereUrl.slice(0, -4);
```

Comment:
Which url we are trying to achieve here?

Reply:
Now I changed this name to "**customisedUrl**". It navigates to MOSIP id-life-cycle managements  [identifiers](https://docs.mosip.io/1.2.0/id-lifecycle-management/identifiers)